### PR TITLE
Angular 9 Update

### PR DESCRIPTION
**Pull request by danielehrhardt**
You should not merge this pull Request but you can check what is necessary that your Project is working with Angular 9.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-    "name": "ionic2-calendar",
-    "version": "0.5.8",
+    "name": "ionic5-calendar",
```
Comment (NiklasMerz) [inline package.json]:
```suggestion
    "name": "ionic2-calendar",
```

I don't think the package name should change for this minor update.

File: package.json [hunk 2]
```diff
@@ -15,25 +15,28 @@
         "url": "git+https://github.com/twinssbc/Ionic2-Calendar.git"
     },
     "scripts": {
-        "build": "rm -rf dist && node_modules/.bin/ngc -p tsconfig.json",
-        "copy_static_files": "cp -r package.json README.md LICENSE tsconfig.json dist/",
+        "build": "rimraf dist && npx ngc -p tsconfig.json && npm run copy_static_files",
+        "copy_static_files": "cpx package.json dist/ &&  cpx README.md dist/ && cpx LICENSE dist/ && cpx tsconfig.json dist/",
```
Comment (NiklasMerz) [inline package.json:19]: Why does this introduce two new dependencies for basic shell commands? Windows support?

File: tsconfig.json [hunk 3]
```diff
@@ -2,12 +2,13 @@
     "compilerOptions": {
         "allowSyntheticDefaultImports": true,
         "allowUnreachableCode": false,
+
```
Comment (NiklasMerz) [inline tsconfig.json]:
```suggestion
```

Remove line

File: tsconfig.json [hunk 4]
```diff
@@ -17,12 +18,15 @@
...
+        "target": "es2015",
+        "typeRoots": ["node_modules/@types"]
     },
-    "files": [
-        "src/index.ts"
-    ],
-    "exclude": [
-        "node_modules"
-    ]
+    "angularCompilerOptions": {
```
Comment (NiklasMerz) [inline tsconfig.json:23]: Looks like this is the stuff necessary for Angular 9/Ivy

**Review by NiklasMerz — COMMENTED** [review]
Thanks @danielehrhardt for sharing. I just checked the code and left some questions and comments. 

I will try these changes later in a project.

File: tsconfig.json (hunk 4 above)
Comment (danielehrhardt) [reply tsconfig.json:23]: Correct.

File: package.json (hunk 2 above)
Comment (danielehrhardt) [reply package.json:19]: Yes, correct. I am working with Windows and to support every Distro we should use node Deps to fix this. 
So now it is working with Mac, Linux and Windows.

**Comment by danielehrhardt** [comment]
Something new here?

**Comment by twinssbc** [comment]
@danielehrhardt Thanks for the PR. I did some test on your branch, but the build failed with below error.

> This likely means that the library (@ionic/angular) which declares IonicModule has not been processed correctly by ngcc, or is not compatible with Angular Ivy. Check if a newer version of the library is available, and update if so. Also consider checking with the library's authors to see if the library is expected to be compatible with Ivy.

I saw enableIvy is set to true in your PR. But I have to set it to false to make it work. I also checked Ionic project config, seems also set it to false. If you could shed any lights on how you make it work, that will be great. Right now, I just publish a new version with Ivy support disabled.

I will close this PR, considering it's not working and contains some tmp commits mixed in.

**Pull request closed without merging**